src/main/java/com/xeeva/catalog/RequisitioningPage.java

```diff
@@ -1,763 +1,764 @@
+		if(getRowsCount>0){
+			TestReporter.assertEquals(getCart_RejectedOrders(), cartValue, 
+					"Found the Rejected order details with Cart no. - ["+ cartValue +"]");
+			TestReporter.assertEquals(getRFQ_RejectedOrders(), RFQValue, 
+					"Found the Rejected order details with Cart no. - ["+ RFQValue +"]");
+			TestReporter.assertEquals(getREQ_RejectedOrders(), REQValue,
+					"Found the Rejected order details with Cart no. - ["+ REQValue +"]");
+		}else{
+			TestReporter.assertTrue(false, "No Records found!!!");
+		}
+
+	}
+
+	/**
+	 * @summary: Method to search for RFQ Cancelled item from Rejected Orders tab.
+	 * @author: Praveen Namburi, @version: Created 26-09-2016
+	 */
+	public void copyRFQCancelledItem(){
+		int evenNum = 0;
+		List<WebElement> RFQStatusList = lblRFQStatus;
+		int getRFQStatusCount = RFQStatusList.size();
+		TestReporter.log("Total Rows in Rejected Orders table: " + getRFQStatusCount);
+		//Iterate even number rows
+		for(int rows=2; rows<=getRFQStatusCount-1;rows++){
+			//To get even numbers rows
+			if(rows % 2 == 0){
+				evenNum = rows;
+				String getRFQStatus = driver.findElement(By.xpath("//table[@class='RecentOrderDetailsGrid ']"
+						+ "/tbody/tr[" + evenNum + "]/td[14]/span")).getText();
+				//Get the RFQ cancelled status from Rejected Orders grid.
+				if(getRFQStatus.trim().contains("Cancelled")){
+					//Click on Copy-item link.
+					driver.findElement(By.xpath("//table[@class='RecentOrderDetailsGrid ']/tbody/"
+							+ "tr[" + evenNum + "]/td[17]/a[1]/i")).jsClick();
+					break;
+				}
+				if(getRFQStatus.trim().contains("Cancelled")) break;  
+			}
+		}
+		driver.manage().timeouts().implicitlyWait(Constants.PAGE_TIMEOUT, TimeUnit.SECONDS);
+	}
+	
+	/**
+	 * @summary: Method to click Cart number with valid REQ Number
+	 * @author praveen namburi, @Version: Created 28-09-2016
+	 * @param comments
+	 */
+	public void clickCartwithREQNumber(){
+		tblRecentOrdersGrid.syncVisible();
+		btnSeeMore.syncEnabled(5);
+		btnSeeMore.click();
+
+		//Checking the rows in each page for REQ Number
+		List<WebElement> getRows = tblMainRecentOrdersGrid;
+		int rowsCount = getRows.size();
+		TestReporter.log("Total rows in RecentOrders Grid table: "+ rowsCount);
+		for(int row=1; row<=rowsCount; row++){
+			pl.isDomComplete(driver);
+			String getRFQStatus = driver.findElement
+					(By.xpath("//*[@id='gvRecentOdersGrid']/tbody/tr["+row+"]/td[3]/span")).getText();
+			TestReporter.logStep("REQNumber is : "+getRFQStatus);
+			if(!getRFQStatus.contains("-")){
+				driver.setPageTimeout(2);
+				driver.findElement(By.xpath(".//*[@id='gvRecentOdersGrid']/tbody/tr["+row+"]/td[1]/a")).click();
+				break;
+			}
+			if(row==rowsCount){
+				driver.findElement(By.partialLinkText(">")).click();
+			}
+		}
+	}
+
+	/**
+	 * @summary This Method captures the Quantity in Requisition Page
+	 * @author  Praveen Varma @date 28/9/16
+	 */
+	public void getQtyNumber(){
+		String quantity = txtQuantityREQ.getAttribute("value");
```

Comment:
Why are we just getting the quantity and not returning it?  There is no point in just logging to the reporter.